stylesheets/pages/products.css

```diff
@@ -62,3 +61,8 @@
   gap: var(--spacing-sm);
   padding: 0 0 var(--spacing-xs) 0;
 }
+/** footer **/
+.page-footer{
+  position:static;
```

Comment:
no need to give position static by default it's static

Reply:
actually I've made a separate file for footer and in that the footer position was absolute so I wanted it to be static in case on this particular page so I gave static. But I'll be updating all the footer style so no worries this will be removed